system/autoware_state_monitor/src/autoware_state_monitor_node/autoware_state_monitor_node.cpp

```diff
@@ -110,35 +112,37 @@ std::string getStateMessage(const AutowareState & state)
 
   throw std::runtime_error("invalid state");
 }
+*/
```

Comment:
Remove?

Reply:
Is it okay to completely delete the code related to the string-type message in original AutowareState.msg?

Reply:
Do you mean this? Then, it's okay!
https://github.com/tier4/autoware_iv_msgs/blob/0477e88d1da64df4497b002c7793f8c619ffba52/autoware_system_msgs/msg/AutowareState.msg#L11

Reply:
yes, Thanks.

Reply:
Fixed at 9905e73

Reply:
Do I need to remove the code related to EmergecyState as well?

Reply:
Thank you :heart: